src/test/java/SignUp.java

```diff
@@ -0,0 +1,77 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import java.time.*;
+
+
+import java.time.format.DateTimeFormatter;
+import java.util.concurrent.TimeUnit;
+
+
+
+
+public class SignUp {
+
+
+    @Test
+    public static void signUpSucessTest() {
+
+        WebDriver driver = new ChromeDriver();
+        driver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern(("yyyyMMddHHmmss"));
+        LocalDateTime myDate = LocalDateTime.now();
+        String localTimeString = formatter.format(myDate);
+        String date1= localTimeString.toString();
+        String com = new String("@gmail.com");
+        String email = new String(date1.concat(com));
+
+        driver.get("https://deens.com/register");
+        WebElement userButton = driver.findElement(By.cssSelector("#username"));
+        userButton.sendKeys(date1);
+        WebElement emailField = driver.findElement(By.cssSelector("#email"));
+        emailField.sendKeys(email);
+        WebElement passwordField = driver.findElement(By.cssSelector("#password"));
+        passwordField.sendKeys("Test123Test123");
+        driver.findElement(By.cssSelector(".ui.large.fluid.button.green-btn.pl-btn")).click();
```

Comment:
Are you sure that it is the best locator? I think it can be a shorter. ".pl-btn"